testsuite/MDAnalysisTests/utils/test_units.py

```diff
@@ -96,6 +100,14 @@ def test_energy(self, quantity, unit1, unit2, ref):
     def test_force(self, quantity, unit1, unit2, ref):
         self._assert_almost_equal_convert(quantity, unit1, unit2, ref)
 
+    @pytest.mark.parametrize('quantity, unit1, unit2, ref', (
+        (1, 'A/ps', 'm/s', 1e-10/1e-12),
+        (1, 'A/ps', 'nm/ps', 0.1),
+        (1, 'A/ps', 'pm/ps', 1e2),
```

Comment:
Can you add some entries to cover Angstrom/microsecond, Angstrom/millisecond, Angstrom/AKMA, Angstrom/femtosecond please?

Just converting between these and A/ps is fine (doesn't need to be all-to-all).

Reply:
Okay I will add the mentioned tests, update AUTHORS and CHANGELOG too. Also, my apologies  for bothering you over the silly codecov return I didn't realise it had to do with my feature branch being behind, I will be more careful from next time. Thanks for the review.